### PR TITLE
New job created from portal to coloredcow website is fixed

### LINKED ISSUE #1349: Details related to New job opportunity created in HR module not available in coloredcow website
**Is your feature request related to a problem? Please describe.**
New job opportunity have to created by the user as per company requirements. When user creates new opportunity it is not available   in  coloredcow website.

Please refer screenshot for reference 

1. Create new job opportunity through HR module

image

image

image

2. Job opportunity not available in colored cow website

image

**Describe the solution you'd like**
When new job opportunity is created through Hr module the details, related to job should be updated in the WordPress page for colored cow website. Hence the details related to new job opportunity will be available in colored cow  website.

### PR DESCRIPTION
**Pull request by Shivam-Samant**
#1349

### CODE REVIEW

File: Modules/Invoice/Resources/views/subviews/create/invoice-details.blade.php [hunk 1]
```diff
@@ -140,17 +140,7 @@
...
-                                if (projectName1 < projectName2) {
-                                    return -1
-                                }
-                                if (projectName1 > projectName2) {
-                                    return 1
-                                }
-                                return 0
-                            });
-                            break;
+                            this.projects = _.orderBy(client.projects, 'name', 'asc');
```
Comment (P4NK4J) [inline Modules/Invoice/Resources/views/subviews/create/invoice-details.blade.php:143]: This doesn't seem to be relevant with the issue ?

**Review by P4NK4J — COMMENTED** [review]
Left one comment, else looks good.

File: Modules/Invoice/Resources/views/subviews/create/invoice-details.blade.php (hunk 1 above)
Comment (Shivam-Samant) [reply Modules/Invoice/Resources/views/subviews/create/invoice-details.blade.php:143]: I used this function before but did not use the array name. So I used this now after a conversation with @GauravGusain98.

Comment (P4NK4J) [reply Modules/Invoice/Resources/views/subviews/create/invoice-details.blade.php:143]: Is this line of code related to the issue which you are trying to resolve? If not, let's add it in a separate ticket

**Review by P4NK4J — APPROVED** [review]

**Pull request merged**